spec/models/ability_spec.rb

```diff
@@ -108,10 +117,12 @@
         should_not be_able_to(:read, section)
         should_not be_able_to(:update, section)
         should_not be_able_to(:destroy, section)
+        should_not be_able_to(:publish, section)
 
         should_not be_able_to(:read, file_set)
         should_not be_able_to(:update, file_set)
         should_not be_able_to(:destroy, file_set)
+        should_not be_able_to(:publish, file_set)
```

Comment:
We shouldn't test "publish" for file_set or section.  The publish action is only available for monographs:
https://github.com/curationexperts/heliotrope/blob/master/config/routes.rb#L18-L22